src/servers/src/mysql/federated.rs

```diff
@@ -433,45 +413,4 @@ mod test {
 +----------------------------------+";
         test(query, expected);
     }
-
-    #[test]
```

Comment:
Sorry, why do we remove this test?

Reply:
Because new `set time_zone = 'UTC'` will be dispose by SQL Stmt, not mysql federated, the relative test has been given in test-intergration